tt_metal/hw/ckernels/wormhole_b0/metal/llk_api/llk_sfpu/ckernel_sfpu_gelu.h

```diff
@@ -9,8 +9,9 @@
 #include "noc_nonblocking_api.h"
 
 #include "sfpi.h"
+#include "sfpu/ckernel_sfpu_gelu.h"
 #include "ckernel_sfpu_cdf.h"
```

Comment:
Minor comment: Is this include "ckernel_sfpu_cdf.h" still needed?

Reply:
Let me take a look later, I believe it's needed.

Reply:
@rtawfik01  ckernel_sfpu_cdf.h is necessary, however sfpi.h and using that namespace is not needed. I believe it is required for GS only since its API for initializing GELU' uses TTI_SFPLOADI. I will fix that in the morning, thanks for the hint!

Reply:
@rtawfik01 Hopefully cleaned up for all three archs.